README.md

```diff
@@ -205,17 +204,17 @@ Example:
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
+👤 ** Omotolani Akinmola **
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- GitHub: [@githubhandle](https://github.com/akinmolaomotlani)
+- Twitter: [@twitterhandle](https://twitter.com/AgbayemiAkinmola)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 👤 **Author2**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- Twi tter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/akinmolaomotolani)
```

Comment:
- [ ] To maintain a professional and clear Readme file, I have a suggestion regarding the author's information. Since you are the only person working on this project, it may be a good idea to remove `Author2` from the file. This will help to avoid any confusion and keep the information concise and accurate.👌